package.json

```diff
@@ -70,39 +60,50 @@
 			"STORE_URL": "http://localhost:8802"
 		}
 	},
+	"lorem": "ipsum",
 	"config": {
 		"wp_org_slug": "storefront",
-		"translate": true
+		"translate": true,
+		"assets": {
+			"js": {
+				"min": "assets/js/**/**/*.min.js assets/js/**/*.min.js assets/js/*.min.js",
+				"src": "assets/js/*.js assets/js/**/*.js assets/js/**/**/*.js"
+			},
+			"css": {
+				"min": "assets/css/**/*.css assets/css/**/**/*.css *.css",
+				"src": "assets/css/**/*.scss assets/css/**/**/*.scss *.scss"
+			}
+		}
 	},
 	"license": "GPL-3.0+",
 	"devDependencies": {
-		"@woocommerce/eslint-plugin": "2.2.0",
-		"@wordpress/browserslist-config": "4.1.3",
-		"@wordpress/env": "5.14.0",
-		"@wordpress/eslint-plugin": "14.6.0",
-		"@wordpress/prettier-config": "2.16.0",
-		"@wordpress/scripts": "22.3.0",
-		"autoprefixer": "10.4.13",
-		"bourbon": "7.2.0",
-		"browserslist": "4.21.4",
-		"concurrently": "7.6.0",
-		"cross-env": "^7.0.3",
-		"eslint-plugin-import": "2.27.5",
-		"github-label-sync": "2.2.0",
-		"jest-puppeteer": "6.1.1",
+		"@woocommerce/eslint-plugin": "2.3.0",
+		"@wordpress/browserslist-config": "6.6.0",
+		"@wordpress/env": "10.6.0",
+		"@wordpress/eslint-plugin": "15.1.0",
+		"@wordpress/prettier-config": "2.25.13",
+		"@wordpress/scripts": "27.2.6",
+		"autoprefixer": "10.4.20",
+		"bourbon": "7.3.0",
+		"browserslist": "4.23.3",
+		"concurrently": "8.2.2",
+		"cross-env": "7.0.3",
+		"eslint-plugin-import": "2.29.1",
+		"github-label-sync": "2.3.1",
+		"jest-puppeteer": "10.1.0",
 		"lodash": "4.17.21",
 		"node-wp-i18n": "1.2.7",
 		"onchange": "7.1.0",
-		"postcss-cli": "9.1.0",
-		"prettier": "npm:wp-prettier@2.6.2",
-		"puppeteer": "14.4.1",
-		"rtlcss": "4.0.0",
+		"postcss-cli": "11.0.0",
+		"prettier": "npm:wp-prettier@2.8.5",
+		"puppeteer": "23.2.1",
+		"rtlcss": "4.3.0",
 		"susy": "2.2.14",
-		"typescript": "5.0.4",
-		"uglify-js": "3.17.4"
+		"typescript": "5.5.4",
+		"uglify-js": "3.19.3"
 	},
 	"engines": {
-		"node": "14.21.1",
+		"node": "^20.11.1",
 		"npm": "9.1.3"
```

Comment:
Let's also pump the npm version too, as v10 is ship with node 20

Reply:
https://nodejs.org/en/about/previous-releases

Reply:
Updated in ea5e7b88f9743727bd81afc5df05af9ba2785931.